File: Programming Basics with Python/03 Conditional statements advanced/More exercises/03_Flowers.py
chrysanthemums = int(input())
roses = int(input())
tulips = int(input())
season = input().lower()
holiday = input().lower()
flowers = chrysanthemums + roses + tulips
chrysanthemum_price = 3.75
rose_price = 4.5
tulip_price = 4.15
if season == "spring" or season == "summer":
    chrysanthemum_price = 2
    rose_price = 4.1
    tulip_price = 2.5
if holiday == "y":
    chrysanthemum_price *= 1.15
    rose_price *= 1.15
    tulip_price *= 1.15
bouquet_price = chrysanthemums * chrysanthemum_price + roses * rose_price + tulips * tulip_price
if tulips > 7 and season == "spring":
    bouquet_price *= 0.95
if roses >= 10 and season == "winter":
    bouquet_price *= 0.9
if flowers > 20:
    bouquet_price *= 0.8
arranging_bouquet = 2
result = bouquet_price + arranging_bouquet
print(f"{result:.2f}")

File: JS Basics/03 Nested Conditional Statements - More Exercises/03. Flowers.js
function shop(chrysanthemums, roses, tulips, season, holiday) {
    chrysanthemums = Number(chrysanthemums);
    roses = Number(roses);
    tulips = Number(tulips);
    let priceChrysanthemums = 0;
    let priceRoses = 0;
    let priceTulips = 0;
    if (season == "Spring" || season == "Summer") {
        priceChrysanthemums = 2 * chrysanthemums;
        priceRoses = 4.1 * roses;
        priceTulips = 2.5 * tulips;
    }
    else {
        priceChrysanthemums = 3.75 * chrysanthemums;
        priceRoses = 4.5 * roses;
        priceTulips = 4.15 * tulips;
    }
    if (holiday == "Y") {
        priceChrysanthemums *= 1.15;
        priceRoses *= 1.15;
        priceTulips *= 1.15;
    }
    let bouquet = priceChrysanthemums + priceRoses + priceTulips;
    if (season == "Spring" && tulips > 7) {
        bouquet *= 0.95;
    }
    if (season == "Winter" && roses >= 10) {
        bouquet *= 0.9;
    }
    if ((roses + tulips + chrysanthemums) > 20) {
        bouquet *= 0.8;
    }
    console.log((bouquet + 2).toFixed(2))
}
shop(10, 10, 10, "Autumn", "N")